Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM gcr.io/google_appengine/base
 # at https://hub.docker.com/r/google/cloud-sdk/~/dockerfile/
 
 ENV \
-	CACHE_BUST=1 \
+	CACHE_BUST=2 \
```

Comment:
The idea here is to invalidate that cache so that the install on Line 41 is fresh, right? Or is there a dependency in the Dockerfile that's using this env var?